lib/web/apiserver.go

```diff
@@ -1602,8 +1602,8 @@ type CreateSessionResponse struct {
 	SessionInactiveTimeoutMS int `json:"sessionInactiveTimeout"`
 }
 
-func newSessionResponse(ctx *SessionContext) (*CreateSessionResponse, error) {
-	accessChecker, err := ctx.GetUserAccessChecker()
+func newSessionResponse(scx *SessionContext) (*CreateSessionResponse, error) {
```

Comment:
It doesn't really matter, but I've encouraged `sctx` to others recently if you want to be consistent.